proto-lens-protoc/src/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -284,13 +284,13 @@ generateMessageDecls fieldModName syntaxType env protoName info =
       $ deriving' ["Prelude.Show", "Prelude.Eq", "Prelude.Ord"]
     | oneofInfo <- messageOneofFields info
     ] ++
-    -- instance HasLens' Foo "foo" Bar
+    -- instance Bar ~ a => HasLens' Foo "foo" a where
     --   lensOf _ = ...
     -- Note: for optional fields, this generates an instance both for "foo" and
     -- for "maybe'foo" (see plainRecordField below).
-    [ uncommented $ instDecl []
+    [ uncommented $ instDecl [equalP "a" (tyParen t)]
```

Comment:
I think this results in `a ~ Bar` rather than `Bar ~ a` in the comment above. Make them consistent?

Reply:
Good catch.  Switched the comment.